app/helpers/WorkersHelpers.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class WorkersHelpers
```

Comment:
Dependency Injection uses a Service Container. You are adding Services to your Container, not Helpers.